en/c-team-codex.md

```diff
@@ -0,0 +1,53 @@
+# C-Team Codex
+
+The C-Team in Applifting is a group of dedicated leaders who guide the company's direction and embody our core values. They ensure Applifting runs smoothly, grows sustainably, and stays true to its teal principles. By living the ideals in this codex, the C-Team drives Applifting towards its mission of creating digital products with positive impact on planetary scale.
+
+## Lead with Vision, Not Control
+
+### Trust in Teal culture
```

Comment:
Here I am missing core teal principles since Trusting in Teal should be the way. IMHO this section should answer what Teal culture IS: Be yourself and encourage others to be themselves, take responsibility for yourself by empowering others to accept their responsibility for themselves - not to take their responsibility on your shoulders! Lastly: listen/look around how the environment is changing and adapt to it.
These are my vaguely described 3 core teal principles, please feel free to rephrase.